tasks/es/ine.py

```diff
@@ -86,7 +86,7 @@ def columns(self):
                   boundary_type['cartographic_boundary']],
         )
         cusec_id = OBSColumn(
-            name=u"Secci\xf3n Censal",
+            name="Secci\xf3n Censal",
```

Comment:
🤦‍♂️

Reply:
We can probably change that to `Sección Censal` now, without having to use the escape characters.

Reply:
Yep, now with p3 we should after this change is merged